ipn/serve.go

```diff
@@ -163,6 +163,31 @@ func (sc *ServeConfig) GetTCPPortHandler(port uint16) *TCPPortHandler {
 	return sc.TCP[port]
 }
 
+// HasPathHandler returns whether if ServeConfig has at least
```

Comment:
Go doc style (sorry) is "reports whether"

(the pedantic argument was always "but the function _always_ returns!")